apps/token/src/components/wallet-card/wallet-card.tsx

```diff
@@ -29,10 +29,14 @@ interface WalletCardProps {
 }
 
 export const WalletCard = ({ dark, children }: WalletCardProps) => {
-  const className = classNames('text-ui border border-white', 'p-8', {
-    'bg-black text-white': dark,
-    'bg-white text-black': !dark,
-  });
+  const className = classNames(
+    'text-ui border border-white',
+    'pt-4 pl-8 pr-12 pb-12',
+    {
+      'bg-black text-white': dark,
```

Comment:
question (non-blocking): do we need this dark property? normally I see the tailwind prefix "dark:xyz-classname". are there cases where we want to override the theme setting and force e.g. a light appearance on a dark theme, or is the context here something completely different?

Reply:
@notbot00 this is an older implementation from TFE that's not been fully removed yet. It'll get done when we add a light theme https://github.com/vegaprotocol/frontend-monorepo/issues/612